sdcm/cluster_gce.py

```diff
@@ -198,9 +195,10 @@ class GCECluster(cluster.BaseCluster):  # pylint: disable=too-many-instance-attr
     Cluster of Node objects, started on GCE (Google Compute Engine).
     """
 
-    def __init__(self, gce_image, gce_image_type, gce_image_size, gce_network, services, credentials,  # pylint: disable=too-many-arguments
-                 cluster_uuid=None, gce_instance_type='n1-standard-1', gce_region_names=None,
-                 gce_n_local_ssd=1, gce_image_username='root', cluster_prefix='cluster',
+    # pylint: disable=too-many-arguments
+    def __init__(self, gce_image, gce_image_type, gce_image_size, gce_network, services: list[NodeDriver],
```

Comment:
can we drop `services` and make it part of GCEProvisioner - so we don't use it explicitly?

See Azure implementation of classes in `cluster_azure.py` - GCE implementation should be very similar (almost identical). So in future we'll completely remove discrepancies between them. Our goal is to drop `GCECluster`, `AzureCluster` etc. in favor of just `Cluster` in future.